test/lib/PlacesTestUtils.js

```diff
@@ -24,22 +26,69 @@ const PlacesTestUtils = Object.freeze({
    *            [optional] visitDate: visit date in microseconds from the epoch
    *            [optional] referrer: nsIURI of the referrer for this visit
    *          }
-   *
-   * @return {Promise}
-   *            resolves When all visits have been added successfully.
-   *            rejects JavaScript exception.
    */
   addVisits: Task.async(function*(placeInfo) {
```

Comment:
Q: does it matter if a consumer sends in undefined or null as palceInfo? Is it worth checking for that?